EngineLayer/CrosslinkSearch/CrosslinkSpectralMatch.cs

```diff
@@ -9,26 +9,34 @@ namespace EngineLayer.CrosslinkSearch
 {
     public class CrosslinkSpectralMatch : PeptideSpectralMatch
     {
-        public CrosslinkSpectralMatch(PeptideWithSetModifications theBestPeptide, int notch, double score, int scanIndex, Ms2ScanWithSpecificMass scan, DigestionParams digestionParams, List<MatchedFragmentIon> matchedFragmentIons)
-            : base(theBestPeptide, notch, score, scanIndex, scan, digestionParams, matchedFragmentIons)
+        public CrosslinkSpectralMatch(PeptideWithSetModifications theBestPeptide, int notch, double score, int scanIndex, Ms2ScanWithSpecificMass scan, CommonParameters commonParameters, List<MatchedFragmentIon> matchedFragmentIons)
+            : base(theBestPeptide, notch, score, scanIndex, scan, commonParameters, matchedFragmentIons)
         {
-            this.XLTotalScore = score;
+            
         }
 
         public CrosslinkSpectralMatch BetaPeptide { get; set; }
+
         public List<int> LinkPositions { get; set; }
-        public double DeltaScore { get; set; }
         public double XLTotalScore { get; set; } //alpha + beta psmCross.
```

Comment:
Where is this value calculated?

Reply:
We keep track of the second best score now. When someone wants the delta score, we return total score minus second best
![image](https://user-images.githubusercontent.com/16841846/72359230-8795aa80-36b3-11ea-9c35-9a6cb01f3793.png)